omod/src/main/java/org/openmrs/module/providermanagement/fragment/controller/PersonNameFragmentController.java

```diff
@@ -14,13 +14,46 @@
 
 package org.openmrs.module.providermanagement.fragment.controller;
 
-import org.openmrs.layout.web.name.NameSupport;
+import java.lang.reflect.Method;
+import java.lang.reflect.InvocationTargetException;
+
 import org.openmrs.ui.framework.fragment.FragmentModel;
 
 public class PersonNameFragmentController {
-
-    public void controller(FragmentModel model) {
-        model.addAttribute("layoutTemplate", NameSupport.getInstance().getDefaultLayoutTemplate());
-    }
-
+	
+	/**
```

Comment:
If we're going to include Javadocs, they should actually provide some information.